tests/unit/intake_rubicon/test_publish.py

```diff
@@ -63,10 +66,12 @@ def test_publish_to_file(project_client):
     project.log_experiment()
     project.log_experiment()
 
-    catalog_yaml = publish(project.experiments(), output_filepath="memory://catalog.yml")
```

Comment:
the memory filesystem started being weird when these tests were run via the edgetest plugin, so I switched it to a `TemporaryDirectory`. the memory filesystem can get weird when things are multi-processed - may be something to do with the way edgetest calls python/pytest